modules/defining-running-jobs.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * /develop/running-kueue-jobs.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="defining-running-jobs_{context}"]
+= Defining a job to run with {product-title}
+
+When you are defining a job to run with {product-title}, ensure that it meets the following criteria:
+
+* Specify the local queue to submit the job to, by using the `kueue.x-k8s.io/queue-name` label.
+* Include the resource requests for each job pod.
+
+{product-title} suspends the job, and then starts it when resources are available. {product-title} creates a corresponding workload, represented as a `Workload` object with a name that matches the job.
+
+.Prerequisites
+
+include::snippets/prereqs-snippet-yaml-user.adoc[]
+* You have identified the name of the local queue that you want to submit jobs to.
+
+.Procedure
+
+. Create a `Job` object.
++
+.Example job
+[source,yaml]
+----
+apiVersion: batch/v1
+kind: Job # <1>
+metadata:
+  generateName: sample-job- # <2>
+  namespace: default
+  labels:
+    kueue.x-k8s.io/queue-name: user-queue # <3>
+spec:
+  parallelism: 3
+  completions: 3
+  template:
+    spec:
+      containers:
+      - name: dummy-job
+        image: registry.k8s.io/e2e-test-images/agnhost:2.53
+        args: ["entrypoint-tester", "hello", "world"]
+        resources: # <4>
+          requests:
+            cpu: 1
+            memory: "200Mi"
+      restartPolicy: Never
+----
+<1> Defines the resource type as a `Job` object, which represents a batch computation task.
+<2> Provides a prefix for generating a unique name for the job.
+<3> Identifies the queue to send the job to.
+<4> Defines the resource requests for each pod.
+
+. Run the job by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+.Verification
+
+* Verify that pods are running for the job you have created, by running the following command and observing the output:
++
+[source,terminal]
+----
+$ oc get job <job-name>
```

Comment:
Would `oc get job <job-name> -w` to watch for status updates?

Reply:
I'm not really sure that's necessary - they could be waiting a while, depending on when Kueue decides to start the job. It depends on resources being available.